storage/src/journal/contiguous/tests.rs

```diff
@@ -128,6 +128,15 @@ where
     assert!(oldest > prev_oldest);
     assert_eq!(oldest, 20);
 
+    // Prune all
+    journal.prune(30).await.unwrap();
+    assert_eq!(journal.oldest_retained_pos().await.unwrap().unwrap(), 30);
```

Comment:
Why expect `Some(30)` instead of `None`?

From the comment on `oldest_retained_pos`:
`Returns None if the journal is empty or if all items have been pruned.`